r2dbc/core/pom.xml

```diff
@@ -103,6 +103,17 @@
       <groupId>io.netty</groupId>
       <artifactId>netty-handler</artifactId>
       <version>4.1.76.Final</version>
+      <exclusions>
+        <exclusion>
+          <!--
+          Because this svn dependency is provided-scope, the users of this
+          library don't get this svn dependency anyway. Explicitly excluding
+          this dependency for the dependencyConvergence enforcer rule.
+          -->
+          <groupId>org.graalvm.nativeimage</groupId>
+          <artifactId>svm</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Since we scoped this to just jdbc/postgres for now, should we revert the changes to other files until you update them?

Reply:
Reverted the changes under r2dbc modules.

Reply:
But they needed to have `svm` elements for dependencyManagement sections.

https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/pull/805/commits/ea0adcc9e42b90efd270b47806337fa59156ca39